daemon/algod/api/server/v2/handlers.go

```diff
@@ -863,7 +866,8 @@ func (v2 *Handlers) WaitForBlock(ctx echo.Context, round uint64) error {
 	select {
 	case <-v2.Shutdown:
 		return internalError(ctx, err, errServiceShuttingDown, v2.Log)
-	case <-time.After(1 * time.Minute):
+	case <-time.After(WaitForBlockTimeout):
+		return notFound(ctx, err, errWaitForBlockTimeout, v2.Log)
```

Comment:
Fixed the documentation.

Yes, if code wasn't handling errors it would break. But there are multiple ways for this function to return errors, so it's likely that people are already handling them.